src/engine/ui/basics/Layout.hxx

```diff
@@ -19,48 +18,4 @@ struct LayoutData
   struct ImFont *font = nullptr;            ///< <internal> font pointer for elements
```

Comment:
If this is all that's in this file, we should probably just move this struct somewhere else and delete `Layout.hxx`. Do you have any ideas as to where we should put this?

Reply:
moveout it to upper folder